cairo-sys-rs/src/lib.rs

```diff
@@ -21,6 +21,7 @@ pub mod enums;
 
 use enums::{
     Status,
+    Content,
```

Comment:
As far I see `cairo_content_t` it just C enum, that has name to separate from other cairo enums.
Rust `Content` is equivalent, so IMHO better way just replace `cairo_content_t` with type alias `pub type cairo_content_t = enums::Content;` and remove this line, this way you don't need do full crate replace of `cairo_content_t`.
In gtk used different method: `KeyFileError` and `GKeyFileError` is completely separated but `KeyFileError` implement `ToGlib<GlibType = ffi::GKeyFileError>`, but `KeyFileError` resides in normal crate, not sys, so it not applied to this case.